src/components/Photo.jsx

```diff
@@ -1,15 +1,19 @@
 import styles from '../styles/photo'
 
 import React from 'react'
+import { Link } from 'react-router'
 
-const STACK_FILES_DOWNLOAD_PATH = 'http://cozy.local:8080/files/download'
+import { STACK_FILES_DOWNLOAD_PATH } from '../constants/config'
 
-const Photo = ({ photo }) => {
+export const Photo = ({ photo, router }) => {
```

Comment:
Thanks @goldoraf. And @enguerran, you're right but even that I prefer to keep the `router` here in order to be very explicit about the router dependency, moreover if this is not a matter for performances we can keep it.